settings.gradle.kts

```diff
@@ -10,6 +10,14 @@ pluginManagement {
   }
 }
 
+dependencyResolutionManagement {
+  repositories {
+    mavenCentral()
+    google()
+    maven("https://maven.pkg.jetbrains.space/public/p/compose/dev/")
```

Comment:
```suggestion
```
What is this for? 1.7.0-rc01 is on Maven Central and the Android variants are all on Google's Maven repo.

Reply:
It resolves the "Could not find org.jetbrains.compose.compiler:compiler:1.5.15."
"1.5.15" version is published to the compose repository only
https://mvnrepository.com/artifact/org.jetbrains.compose.compiler/compiler?repo=space-public-compose-dev